### ISSUE TITLE
v3.26.0 crash: `java.lang.IllegalStateException: FragmentManager has been destroyed`

### ISSUE THREAD

**Issue by mlsxlist** [nextcloud/android#11972]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

Don't know how to recreate.

### Expected behaviour

Should not crash

### Actual behaviour

It crashed

### Android version

Android 13

### Device brand and model

Samsung Galaxy S23 Ultra

### Stock or custom OS?

Stock

### Nextcloud android app version

3.26.0

### Nextcloud server version

25.0.11

### Using a reverse proxy?

No

### Cause of error
```java
Exception in thread "main" java.lang.IllegalStateException: FragmentManager has been destroyed
    at androidx.fragment.app.FragmentManager.enqueueAction(FragmentManager.java:1666)
    at androidx.fragment.app.BackStackRecord.commitInternal(BackStackRecord.java:341)
    at androidx.fragment.app.BackStackRecord.commit(BackStackRecord.java:306)
    at androidx.fragment.app.DialogFragment.show(DialogFragment.java:524)
    at com.owncloud.android.ui.activity.SyncedFoldersActivity.onSyncFolderSettingsClick(SyncedFoldersActivity.kt:595)
    at com.owncloud.android.ui.activity.SyncedFoldersActivity$load$1$1.invokeSuspend(SyncedFoldersActivity.kt:333)
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
    at android.os.Handler.handleCallback(Handler.java:942)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loopOnce(Looper.java:226)
    at android.os.Looper.loop(Looper.java:313)
    at android.app.ActivityThread.main(ActivityThread.java:8762)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30260090`
* Build flavor: `gplay`

### Device information
* Brand: `samsung`
* Device: `dm3q`
* Model: `SM-S918B`
* Id: `TP1A.220624.014`
* Product: `dm3qxeea`

### Firmware
* SDK: `33`
* Release: `13`
* Incremental: `S918BXXU3AWGJ`

**Comment by joshtrichards** [comment]
What were you doing/trying to do in the app at the time you saw the exception?

**Comment by joshtrichards** [comment]
Fixed in #13075
